src/JsonSchema/Constraints/Factory.php

```diff
@@ -82,6 +87,23 @@ public function __construct(
     }
 
     /**
+     * Get package files
+     *
+     * @param string File path, relative to the root of the package
+     */
+    public function getPackageFile($path)
```

Comment:
@bighappyface @shmax Is this an acceptable solution to the getting-package-files thing, or would you rather keep things simpler?